portfolio.js

```diff
@@ -23,3 +23,28 @@ projectInfo.forEach(function(currentProject) {
 projects.forEach(function(project) {
   $('#projects').append(project.toHtml());
 });
```

Comment:
You will want to either move this into a view function (like renderIndexPage in our lab) or put it under line 30 so that our projects are getting rendered after they are being instantiated and pushed into the array. It's preferable to modularize our code and abstract this functionality into a portfolioView method.